appveyor.yml

```diff
@@ -0,0 +1,9 @@
+version: '4.5.4-SNAPSHOT+AppVeyor.{build}'
+os: Windows Server 2012
+build_script:
+  - mvn compile -q --batch-mode 
```

Comment:
Yes, Maven is [available in AppVeyor](https://www.appveyor.com/docs/installed-software/#java).